manual/src/main/asciidoc/include/introduction.adoc

```diff
@@ -421,3 +421,46 @@ in SonarQube:
 ----
 
 NOTE: The first column will be used automatically if no primary column is explicitly specified.
+
+==== Baseline Management
```

Comment:
This would also allow me to run the analysis, create an initial baseline and then remove specific results that should not be baselined. With that, I can ignore only specific findings. Maybe this should be documented

Reply:
Correct, this is intended and the only reason, why the columns are in the baseline file (there’s no technical reason). Will add it to the documentation.